packages/api-specification/api/system.ts

```diff
@@ -0,0 +1,88 @@
+import {Bounds, Size} from "./common";
+
+export interface SystemAPI {
+    /**
+     * Info about all displays available

```

Comment:
other interface needed is for screen sharing... https://symphonyoss.atlassian.net/wiki/display/WGDWAPI/getMediaSources+API which is essentially equal to https://electron.atom.io/docs/api/desktop-capturer/#desktopcapturer